### PR TITLE
Update dependencies using the new Signall::builder API

### PR DESCRIPTION
**Pull request by thiblahute**

### CODE REVIEW

File: plugins/src/webrtcsink/imp.rs [hunk 1]
```diff
@@ -2497,43 +2494,36 @@ impl ObjectImpl for WebRTCSink {
...
+                    .build(),
                 /*
                  * RsWebRTCSink::get_consumers:
                  *
                  * List all consumers (by ID).
                  */
-                glib::subclass::Signal::builder(
-                    "get-consumers",
-                    &[],
-                    <Vec<String>>::static_type().into(),
```
Comment (sdroege) [inline plugins/src/webrtcsink/imp.rs:2517]: You still need to provide the parameter types

**Review by sdroege — COMMENTED** [review]
Looks good otherwise, thanks!

**Comment by sdroege** [comment]
Also best to wait for https://github.com/gtk-rs/gtk-rs-core/pull/734#discussion_r946469067

**Comment by MathieuDuponchelle** [comment]
@sdroege your comment is addressed here right?

**Comment by sdroege** [comment]
No

**Comment by MathieuDuponchelle** [comment]
ah my bad

**Comment by thiblahute** [comment]
> Also best to wait for https://github.com/gtk-rs/gtk-rs-core/pull/734#discussion_r946469067

This hasn't been done yet afaiu

**Comment by MathieuDuponchelle** [comment]
> This hasn't been done yet afaiu. Other comments should be addressed

OK, please just merge when done then :)

**Comment by sdroege** [comment]
https://github.com/gtk-rs/gtk-rs-core/pull/748 is pending, then this can go in

**Comment by sdroege** [comment]
That one is merged now

**Comment by thiblahute** [comment]
Updated, should be good to go.

**Comment by MathieuDuponchelle** [comment]
thanks :)

**Pull request merged**